src/test/resources/network/brightspots/rcv/test_data/tiebreak_test_TODO/tiebreak_interactive_sequential_config.json

```diff
@@ -0,0 +1,57 @@
+{
+  "outputSettings": {
+    "contestName": "Tiebreak test",
+    "outputDirectory": "output",
+    "contestDate": "2017-12-03",
+    "contestJurisdiction": "Funkytown, USA",
+    "contestOffice": "Sergeant-at-Arms",
+    "tabulateByPrecinct": false,
+    "generateCdfJson": false
+  },
+  "cvrFileSources": [
+    {
+      "filePath": "tiebreak_test_cvr.xlsx",
+      "firstVoteColumnIndex": 2,
+      "firstVoteRowIndex": 2,
+      "provider": "ES&S"
+    }
+  ],
+  "candidates": [
+    {
+      "name": "Mookie Blaylock",
+      "excluded": false
+    },
+    {
+      "name": "Yinka Dare",
+      "excluded": false
+    },
+    {
+      "name": "George Gervin",
+      "excluded": false
+    },
+    {
+      "name": "Sedale Threatt",
+      "excluded": false
+    }
+  ],
+  "rules": {
+    "tiebreakMode": "interactive",
+    "overvoteRule": "alwaysSkipToNextRank",
+    "numberOfWinners": 2,
+    "decimalPlacesForVoteArithmetic": 4,
+    "minimumVoteThreshold": 0,
+    "maxSkippedRanksAllowed": 0,
+    "maxRankingsAllowed": 3,
+    "sequentialMultiSeat": true,
```

Comment:
why did you pick this complicated setup for your test?

Reply:
I just duplicated the existing tiebreak* test and switched it to sequentialMutliSeat true so it would explore that branch of the code. This was necessary for making sure the cancellation worked in every case, and figured we might as well keep it around for better coverage.